src/prop/prop_proof_manager.cpp

```diff
@@ -96,9 +132,8 @@ std::vector<std::shared_ptr<ProofNode>> PropPfManager::getProofLeaves(
   Assert(pc == modes::ProofComponent::THEORY_LEMMAS
          || pc == modes::ProofComponent::PREPROCESS);
   std::vector<std::shared_ptr<ProofNode>> pfs =
-      pc == modes::ProofComponent::THEORY_LEMMAS
-          ? d_proofCnfStream->getLemmaClausesProofs()
-          : d_proofCnfStream->getInputClausesProofs();
+      pc == modes::ProofComponent::THEORY_LEMMAS ? getLemmaClausesProofs()
```

Comment:
ditto

Reply:
This is necessary since the called method iterates and adds the proofs to a vector.